jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/src/main/resources/case_definitions.json

```diff
@@ -4,7 +4,10 @@
     "name": "Example Case for Development",
     "containerId": "example",
     "roles": {
-      "participant" : -1
```

Comment:
role owner is also returned from the kie-server and needs to be excluded, added this in here: https://github.com/kiegroup/jbpm-wb/pull/652/files#diff-e70320980c3b40bbe1c0b856e9fae313R7